packages/terra-status/src/Status.jsx

```diff
@@ -16,8 +18,8 @@ const propTypes = {
 };
 
 const Status = ({ color, children, ...customProps }) => {
-  const statusClassNames = classNames(
-    'terra-Status',
+  const statusClassNames = cx(
+    'status',
     { [`${customProps.className}`]: customProps.className },
```

Comment:
this can just be `customProps.className`.. if it isn't present, classNames won't render it out